src/definition/guessfunc.go

```diff
@@ -0,0 +1,15 @@
+package definition
+
+import (
+	"fmt"
+	"strings"
+)
+
+// K8sMetricSetTypeGuesser is the metric set type guesser for k8s integrations.
+func K8sMetricSetTypeGuesser(_, groupLabel, _ string, _ RawGroups) (string, error) {
```

Comment:
by the way as far as I understand the weird signature is due to:
```
type GuessFunc func(clusterName, groupLabel, entityID string, groups RawGroups) (string, error)
```

Could we change that to have just the parameter used?

Reply:
You actually addressed it in #581, thanks for simplifying 🚀